policies/sdc_approve.rego

```diff
@@ -0,0 +1,26 @@
+package karavi.sdc.approve
```

Comment:
are there plans to add this new sdc-approve policy to the helm-charts and csm operator authorization deployments?

Reply:
Done for csm-operator. For helm-charts, it is under review